src/PlaywrightSharp.Tests/BaseTests/PlaywrightSharpLoader.cs

```diff
@@ -11,12 +11,29 @@ public class PlaywrightSharpLoader
     {
         internal static SimpleServer Server { get; private set; }
         internal static SimpleServer HttpsServer { get; private set; }
+        internal static string ContentRoot { get; private set; }
 
         internal static async Task SetupAsync()
         {
-            var downloaderTask = TestConstants.GetNewBrowserType().CreateBrowserFetcher().DownloadAsync();
-            Server = SimpleServer.Create(TestConstants.Port, TestUtils.FindParentDirectory("PlaywrightSharp.TestServer"));
-            HttpsServer = SimpleServer.CreateHttps(TestConstants.HttpsPort, TestUtils.FindParentDirectory("PlaywrightSharp.TestServer"));
+            string host = !TestConstants.IsChromium ? TestConstants.BROWSER_CDN : null;
+            var browserFetcher = TestConstants.GetNewBrowserType().CreateBrowserFetcher(new BrowserFetcherOptions
+            {
+                Host = host
+            });
+            int percentage = 0;
+            browserFetcher.DownloadProgressChanged += (sender, e) =>
+            {
+                if (percentage != e.ProgressPercentage)
+                {
+                    percentage = e.ProgressPercentage;
+                    Console.WriteLine($"[{TestConstants.Product}] downloading browser {percentage}%");
+                }
+            };
+            Console.WriteLine($"Downloading browser from {host}...");
+            var downloaderTask = browserFetcher.DownloadAsync();
+            ContentRoot = TestUtils.FindParentDirectory("PlaywrightSharp.TestServer");

```

Comment:
Why do we need this?

Reply:
for `GoToTests.ShouldOverrideReferrerPolicy`

Reply:
See my comment on the test.